Header header
float32[] data
float32[] wavelengths
int32 integrationTime
int32 lampPower
int32 humidity
int32 temp
